pkgs/development/libraries/criterion/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchgit, cmake, pkg-config, boxfort, libcsptr, dyncall, nanomsg }:
+
+stdenv.mkDerivation rec {
+  pname = "criterion";
+  version = "2.3.3";
+
+  src = fetchgit
+    { url = "git://github.com/Snaipe/Criterion";
+      rev = "v${version}";
+      fetchSubmodules = true;
+      sha256 = "0y1ay8is54k3y82vagdy0jsa3nfkczpvnqfcjm5n9iarayaxaq8p";
+    };
+
+  buildInputs =
+    [ cmake
+      pkg-config
+      boxfort
+      libcsptr
+      dyncall
+      nanomsg
+    ];
+
+  # CMake needs a little help to find BoxFort and pkg-config
```

Comment:
Is this a problem with `BoxFort`? If a fix for this can be applied in `BoxFort` instead of here that would be more desirable as every package/developer on nix using `BoxFort` conceivably has to implement this workaround.

Reply:
Seeing as `pkg-config` goes undetected the same way, I don't think it's a `boxfort` problem.

Reply:
So, is it resolved or no?

Reply:
I've seen i where some people will just see if a directory exists, instead of using the `find_packge` command. I think this is alright, but it would probably be good to open a ticket upstream

Reply:
Will do. I'm assuming all reqs are cleared and we can merge, right? It still says "changes requested"

Reply:
I notice the other PR didn't have need to add this `.third-party` directory.

Reply:
I suspect their modification of `$LD_LIBRARY_PATH` fixed it on their side, I tested it locally and it works